swampmachine-blueprints/src/test/java/net/kiberion/swampmachine/loaders/ModelDescriptorsLoadTest.java

```diff
@@ -37,7 +38,7 @@ public void init() {
     }
 
     
-    //Fails due to CommonViewInfoLoader not setting the imageID
+    @Ignore("Fails due to CommonViewInfoLoader not setting the imageID")
```

Comment:
Can you add the ToDo FixMe comment to this one? Shouldn't be a difficult fix, so I should look at it it nearby future.